### ISSUE TITLE
Add the Slack application to your repository and a Slack channel to improve collaboration

### ISSUE THREAD

**Issue by bill-ramos-rmoswi** [wiltondb/wiltondb#19]
Hi Alex,
Can you please consider integrating your repository for WiltonDB with Slack so that we can make it easier to communicate ideas and suggestions with Slack?
Thanks!
Bill Ramos

BTW, I've created a new repo - https://github.com/bill-ramos-rmoswi/babelfish_postgresql_towel as the Hitchhikers Guide to Babelfish for PostgreSQL. Just a readme as of now, scripts are on the way. :-)

**Comment by staticlibs** [comment]
Hi Bill,

I've created a Slack account for WiltonDB and subscribed there this repo. Never used Slack before - hopefully I've done it correctly. When I tried to send and invite - it generated this [invitation link](https://join.slack.com/t/wiltondbworkspace/shared_invite/zt-2l1kkbook-i1cMw4pjj211XTQ3wsvahQ).

I think the Towel repo is a good idea, for example there is some useful content in this [active discussions thread](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/discussions/15) that can be added to the repo.
